src/Projection/Todo/TodoProjector.php

```diff
@@ -47,4 +48,17 @@ public function onTodoWasPosted(TodoWasPosted $event)
             'status' => $event->todoStatus()->toString()
         ]);
     }
+
+    /**
+     * @param TodoWasMarkedAsDone $event
+     */
+    public function onTodoWasMarkedAsDone(TodoWasMarkedAsDone $event)
+    {
+        $this->connection->update(Table::TODO, [
+            'status' => $event->newStatus()->toString()
+        ],
```

Comment:
should be

    $this->connection->update(Table::TODO, 
        [
            'status' => $event->newStatus()->toString()
        ],
        [
            'id' => $event->todoId()->toString()
        ]
    );